dask/dataframe/groupby.py

```diff
@@ -1633,7 +1632,10 @@ def last(self, split_every=None, split_out=1):
             token="last", func=M.last, split_every=split_every, split_out=split_out
         )
 
-    @derived_from(pd.core.groupby.GroupBy)
+    @derived_from(
+        pd.core.groupby.GroupBy,
+        inconsistencies="If the group is not present, Dask will return an empty Series/DataFrame.",
```

Comment:
We didn't raise a `KeyError` if the group wasn't present even before this PR.

Reply:
Interesting, I'm surprised by that. Do you think we should?

Reply:
As we discussed, this might be out of scope for this PR. :)